digdag-standards/src/main/java/io/digdag/standards/command/ecs/EcsClientConfigBuilder.java

```diff
@@ -0,0 +1,140 @@
+package io.digdag.standards.command.ecs;
+
+import com.google.common.base.Optional;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class EcsClientConfigBuilder
+{
+    private String clusterName;
+    private String launchType;
+    private String accessKeyId;
+    private String secretAccessKey;
+    private String region;
+    private Optional<List<String>> subnets;
+    private int maxRetries;
+    private Optional<String> capacityProviderName;
+    private Optional<Integer> cpu;
+    private Optional<Integer> memory;
+
+    public EcsClientConfig build()
+    {
+        return new EcsClientConfig(this);
+    }
+
+    public EcsClientConfigBuilder withClusterName(String clusterName)
+    {
+        this.clusterName = clusterName;
+        return this;
+    }
+
+    public EcsClientConfigBuilder withLaunchType(String launchType)
+    {
+        this.launchType = launchType;
+        return this;
+    }
+
+    public EcsClientConfigBuilder withAccessKeyId(String accessKeyId)
+    {
+        this.accessKeyId = accessKeyId;
+        return this;
+    }
+
+    public EcsClientConfigBuilder withSecretAccessKey(String secretAccessKey)
+    {
+        this.secretAccessKey = secretAccessKey;
+        return this;
+    }
+
+    public EcsClientConfigBuilder withRegion(String region)
+    {
+        this.region = region;
+        return this;
+    }
+
+    public EcsClientConfigBuilder withSubnets(Optional<String> subnets)
+    {
+        if (subnets.isPresent()) {
+            this.subnets = Optional.of(Arrays.asList(subnets.get().split(",")));
+        }
+        else {
+            this.subnets = Optional.absent();
+        }
```

Comment:
Subnets is not necessary for some use cases. So I defined this files as Optional.